chaco/plot_containers.py

```diff
@@ -19,27 +19,25 @@
     Array,
     Either,
     Enum,
-    Float,
     Instance,
     List,
     Property,
-    String,
+    Str,
     Trait,
     Tuple,
     Int,
 )
-from enable.simple_layout import (
-    simple_container_get_preferred_size,
-    simple_container_do_layout,
-)
+from enable.api import Container, OverlayContainer
+from enable.stacked_container import HStackedContainer, VStackedContainer
+from enable.stacked_layout import stack_layout, stacked_preferred_size
 
 try:
     from enable.api import ConstraintsContainer
 except ImportError:
     ConstraintsContainer = None
 
 # Local relative imports
-from .base_plot_container import BasePlotContainer
+from .plot_component import DEFAULT_DRAWING_ORDER
```

Comment:
we might want to move `DEFAULT_DRAWING_ORDER` somewhere else/somewhere better e.g. `chaco.base` instead of leaving it in `chaco.plot_component`